src/themes/skeleton/styles.css

```diff
@@ -1,199 +1,278 @@
+  --font-size-h2: calc(var(--font-size-body) * 1.563 * 1px);
+  --font-size-h3: calc(var(--font-size-body) * 1.25 * 1px);
+  --font-size-h4: calc(var(--font-size-body) * 1 * 1px);
+  --font-size-h5: calc(var(--font-size-body) * 0.8 * 1px);
+  --font-size-h6: calc(var(--font-size-body) * 0.725 * 1px);
+  --font-size-90: 90%;
+  --font-size-80: 80%;
+  --font-size-75: 75%;
+  --line-height: 1.35;
+
+  /* Theme Colors */
+  --color-stock: #fff;
+  --color-key: #363636;
+
+  /* Primary: Used for primary tone of the theme, for buttons, links, and other elements */
+  --color-primary-100: var(--color-key);
+  --color-primary-500: var(--color-key);
+  --color-primary-700: var(--color-key);
+
+  /* Neutral: For backgrounds and borders.
+  * Note that there are 5 variables, instead of the usual 3. */
+  --color-neutral-100: #ededed;
+  --color-neutral-200: #e1e1e1;
+  --color-neutral-300: #d3d3d3;
+  --color-neutral-400: #c4c4c4;
+  --color-neutral-500: #b3b3b3;
+  --color-neutral-600: #a0a0a0;
+  --color-neutral-700: #898989;
+  --color-neutral-800: #6c6c6c;
+  --color-neutral-900: #3f3f3f;
+
+  color: var(--color-key);
+  font-family: var(--font-family-body);
+  font-size: var(--font-size-body);
+  line-height: var(--line-height);
+  margin-left: auto;
+  margin-right: auto;
+  width: 90%;
+
+  @media (--mq-md) {
+    width: 100%;
+  }
+
+  /* Provide a more intuitive sizing behaviour
+  * https://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+  box-sizing: border-box;
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: inherit;
+  }
+
+  /* To allow media elements to break out of the article width constraints, we first constrain all elements to the
+  * `--max-width-media` size, and then limit non-Media types to the smaller `--max-width` limit */
+  & > *,
+  & > :--MediaTypes,
+  & > :--CodeTypes {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 100%;
+    width: auto;
+
+    @media (--mq-md) {
+      max-width: var(--max-width-media);
+    }
+  }
+
+  & > *:not(:--MediaTypes) {
+    @media (--mq-md) {
+      max-width: var(--max-width);
+    }
+  }
+
+  & > span {
+    display: block;
+  }
+
+  & > :--MediaTypes {
+    margin-bottom: var(--spacer-lg);
+    margin-top: var(--spacer-lg);
+  }
+
+  a,
+  :--Link {
+    &,
+    &:visited {
+      color: inherit;
+    }
+
+    &:hover,
+    &:active {
+      color: var(--color-primary);
+    }
+  }
+
+  ::selection {
+    background-color: var(--color-neutral-300);
+  }
+}
+
+/* This is a top level article, as opposed to Articles which can often be found, amongst other places, in list of references */
+:--Article:--root {
+  padding: var(--spacer-lg) 0;
+
+  & > :--authors,
+  & > :--affiliations {
+    padding-left: 0;
+    list-style-position: inside;
+
+    @media (--mq-md) {
+      list-style-position: outside;
+    }
+  }
+}
+
+:--root > :--CodeBlock:not(stencila-code-chunk),
+:--root > :--CodeChunk:not(stencila-code-chunk) {
```

Comment:
I don't understand the point of either of these selectors: a `CodeBlock` should never be encoded as a `stencila-code-chunk` and a `CodeChunk` should always be encoded as one. So this could be simplified to just `:--root > :--CodeBlock`